tech/languages/php/php-installation.md

```diff
@@ -0,0 +1,59 @@
+---
+title: PHP
+page: php
+section: tech-languages
+---
+
+## PHP
+
+### PHP Installation
+
+To install PHP, simply type:
+
+```
+# dnf install php-cli
+```
+
+Above command will install latest stable PHP package, which includes the command-line interface executing PHP scripts, /usr/bin/php, and the CGI interface. 
+
+
+You might want to install some other packages, e.g. phpunit for unit tests or composer to manage dependencies of PHP projects. You can install them with:
```

Comment:
Use backticks around names of binaries such as phpunit and composer. Or use their proper name: Composer (capital C).